### ISSUE TITLE
[Feature] Passwords

### ISSUE THREAD

**Issue by serialame** [NullCode13/NullRAT#17]
Here I am againnnnnnnn

Add Chrome Password Grabbing

There's a lot of repos on Github about it, Im also certain you could code it yourself at this point.

Bnt

**Comment by NullCode1337** [comment]
ayo sup
idea's not bad, ill try to add it

**Comment by NullCode1337** [comment]
aight gonna finally work on it tr

**Comment by pancak3boy** [comment]
pls add it

**Comment by i6c** [comment]
You can't add it yourself? There is so much repositories that do this @pancak3boy @bntr00t

Or do you not know how to program , and you install free hacker tools on Github ) 😅

**Comment by serialame** [comment]
> You can't add it yourself? There is so much repositories that do this @pancak3boy @bntr00t
> 
> Or do you not know how to program , and you install free hacker tools on Github ) 😅


i dont think you know how this works, this is a request for him to add this. As i said this isn't too  hard to make and if you check my profile for 1 second you might see that i've already made this stuff. Also yes some people do not know how to code and these Repos are made for them to learn or just for them to use the program. Please do not add such useless comennts, thank you.
